packages/bratus-app/src/components/ComponentNode/ComponentNode.jsx

```diff
@@ -98,6 +98,14 @@ const ComponentNode = (node) => {
   const layoutSourceHandlePosition =
     treeLayoutDirection === 'LR' ? 'right' : 'bottom';
 
+  const truncateNodeName = (nodeName, nameLength) => {
+    return nodeName.length > nameLength
+      ? nodeName.slice(0, nameLength - 1).concat('...')
+      : nodeName;
+  };
+
+  const truncatedNodeName = truncateNodeName(node.data.label, 13);
```

Comment:
Nice and easy.
Should the value 13 maybe be placed in the constants file? 
truncatedNodeLength , or similar? :)

Reply:
For sure! I'll do that ☺️

Reply:
Resolved